ddtrace/_trace/span.py

```diff
@@ -549,16 +549,10 @@ def set_exc_info(self, exc_type, exc_val, exc_tb):
             return
 
         self.error = 1
-        self._set_exc_tags(exc_type, exc_val, exc_tb)
-
-    def _set_exc_tags(self, exc_type, exc_val, exc_tb):
```

Comment:
We no longer use `_set_exc_tags` in `opentelemetry/_span.py`.  So we can get rid of this internal method.